ParisBNF/abb/BNFabb20.xml

```diff
@@ -33,12 +33,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     </msIdentifier>
                     <msContents> 
                         <msItem xml:id="ms_i1">
-                            <locus from="3r" to="125v"/>
+                            <locus from="3r" to="127v"/>
                             <title type="complete" ref="LIT3171Commen"/>
-                        </msItem>
-                        <msItem xml:id="ms_i2">
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="3r" to="125v"/>
+                            <title type="complete">Commentary on the Hebrews by John Chrysostom</title>
+                       </msItem>
+                        <msItem xml:id="ms_i1.2">
                             <locus from="126r" to="127v"/>
-                            <title>A notice on St. John Chrysostom taken from the Tārika Walda ʾAmid, the Sənkəssār, Abušākər and the Chronicle of Yoḥannəs Madabbar.</title>
+                            <title type="complete" ref="LIT3171Commen#Vita">Biography</title>
```

Comment:
please write out `Biography of John Chrysostom` - or delete the word Biography, then the label from `LIT3171Commen#Vita` (`Life of John Chrysostom`) will be printed